src/pages/15Years/15Years.vue

```diff
@@ -52,12 +53,22 @@ export default {
 	metaInfo: {
 		title: '15 Years'
 	},
+	inject: ['apollo'],
 	data() {
 		return {
 			headerTheme: fifteenYearHeaderTheme,
 			footerTheme: fifteenYearFooterTheme,
+			expGrow166Aug2020: getExperimentVersion('intercom_messenger', this.apollo) === 'shown',
```

Comment:
In admin, the experiment key could be 'EXP-GROW-166-Aug2020' instead of 'intercom_messenger'